.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+
+---
+
+<!--
+YOUR ISSUE MAY BE CLOSED IF YOU DO NOT FOLLOW THIS TEMPLATE
+
+Consider searching for similar issues before submitting yours.
+-->
+
+**Feature Description:** <!-- A description of the feature you are requesting. -->
```

Comment:
Add the sections: Reasoning, and specific implementation suggestions.

Reply:
Im not sure these are necessary. Especially when considering direct Discord features, is the reasoning not always "Because I want to use it" ?

Reply:
Consider it like SCRUM stories. Put it into perspective for why we should prioritize its addition. Additionally, if we are on top of api updates, feature requests should mainly be abstraction related. Additionally specific implementation suggestions can be helpful as it allows the user to contribute to the implementation without writing more than pseudo code. And multiple opinions on an implementation can lead to good things.